### DISCUSSION TITLE
1.5.0-beta4

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#359] [Обновления]
⚠ Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений
- Улучшено определение языка при отсутвии субтитров на YouTube (убрано использование имени автора каанала) от @SashaXser в #355
- Добавлена возможность загрузки субтитров от Яндекса в формате json (#352, #335)
- Добавлена поддержка rumble.com (#94, #123)
- Фикс работы в Safari (#351)
- Значения lectureData, videoURL и subtitlesURL, теперь, показываются и без дебаг режима (udemy, coursera)
- В полезные ссылки в README добавлены пометки `vot-cli` и `cloudflare`, дабы число вопросов с ними сократилось
- Сайты, с которыми могут возникать ошибки CSP, были помечены в README как требующие доп. действий, дабы число вопросов с ними сократилось
- Добавлено проксирование стримов, чтобы не нужно было использовать отдельное расширение для отключения CORS
- HLS.js обновлен до актуальной версии

## What's Changed
* Протестирована поддержка violentmonkey by @Spp595 in https://github.com/ilyhalight/voice-over-translation/pull/338
* Добавить языковые значки by @szepeviktor in https://github.com/ilyhalight/voice-over-translation/pull/342
* Доработка определение языка при отсутвии субтитров  by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/355

## New Contributors
* @Spp595 made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/338
* @szepeviktor made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/342

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0-beta3...1.5.0-beta4

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0-beta4'>1.5.0-beta4</a>.</em>

**Comment by NullVerdict** [comment]
После отключения перевода стрима, запросы продолжают идти...
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/d309515a-0917-45f0-bd43-dd393d2816c8)
